tools/mk/Makefile.targ

```diff
@@ -122,6 +122,10 @@ check-bash: $(BASH_FILES:%=%.bashchk) $(BASH_FILES:%=%.bashstyle)
 %.bashstyle: %
 	$(BASHSTYLE) $^
 
+.PHONY: check-eslint-fix
```

Comment:
we have a `make fix-style` that reprints everything with `prettier`, probably more accurate than ESLint fix: https://github.com/restify/node-restify/blob/master/tools/mk/Makefile.targ#L137

Reply:
Oh nice! I completely forgot you added that! I'll remove this target  👍